packages/forms/components/TextInput.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from "react";
+import { cx } from "emotion";
+import { inputAppearances, inputContainer, inputValidation } from "../style";
+import {
+  display,
+  flex,
+  flexItem,
+  flush,
+  inputReset,
+  margin,
+  padding,
+  textWeight
+} from "../../shared/styles/styleUtils";
+
+export enum TextInputAppearance {
+  Standard = "standard",
+  Error = "error",
+  Success = "success"
+}
+
+export interface TextInputProps extends React.HTMLProps<HTMLInputElement> {
```

Comment:
is `extends React.HTMLProps<HTMLInputElement>` the best way to ensure we support all the same props that `<input>` does?

Reply:
Yea, I should actually update my Button PR to do something similar :)